### PR TITLE
Update intune-endpoints.md

### PR DESCRIPTION
**Pull request by msbemba**
Added related topics per request

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger8** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1272

File: memdocs/intune/fundamentals/intune-endpoints.md [hunk 1]
```diff
@@ -134,3 +134,14 @@ Intune leverages Google Firebase Cloud Messaging (FCM) for push notification to
...
+
+
+## Related topics
+
+[Office 365 URLs and IP address ranges](/microsoft-365/enterprise/urls-and-ip-address-ranges)
+[Client connectivity](https://support.office.com/article/client-connectivity-4232abcf-4ae5-43aa-bfa1-9a078a99c78b)
+[Content delivery networks](https://support.office.com/article/content-delivery-networks-0140f704-6614-49bb-aa6c-89b75dcd7f1f)
+[Additional endpoints not included in the Office 365 IP Address and URL Web service](additional-office365-ip-addresses-and-urls.md)
+[Managing Office 365 endpoints](managing-office-365-endpoints.md)
+
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/intune-endpoints.md:146]:
```suggestion
[Office 365 URLs and IP address ranges](/microsoft-365/enterprise/urls-and-ip-address-ranges)

[Microsoft 365 network connectivity overview](https://support.office.com/article/client-connectivity-4232abcf-4ae5-43aa-bfa1-9a078a99c78b)

[Content delivery networks (CDNs)](https://support.office.com/article/content-delivery-networks-0140f704-6614-49bb-aa6c-89b75dcd7f1f)

[Other endpoints not included in the Office 365 IP Address and URL Web service](/microsoft-365/enterprise/additional-office365-ip-addresses-and-urls)

[Managing Office 365 endpoints](/microsoft-365/enterprise/managing-office-365-endpoints)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Smritib17 Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Smritib17 Gentle reminder

**Review by Smritib17 — APPROVED** [review]

**Comment by Smritib17** [comment]
#sign-off

**Pull request merged**